pkg/restore/db.go

```diff
@@ -159,11 +160,17 @@ func (db *DB) CreateTable(ctx context.Context, table *utils.Table) error {
 		default:
 			alterAutoIncIDFormat = "alter table %s.%s auto_increment = %d;"
 		}
+
+		// auto inc id overflow
+		autoIncID := table.Info.AutoIncID
+		if table.Info.AutoIncID < 0 {
+			autoIncID = math.MaxInt64
```

Comment:
what about `bigint unsigned` columns?

```sql
create table a (id bigint unsigned primary key auto_increment);
insert into a values (12345678901234567890);
```

Reply:
I was wondering that the allocator can work well 🤔️，it set the allocator with unsigned=false

Reply:
How about add `globalAutoID -= 1` after https://github.com/pingcap/br/blob/master/pkg/backup/client.go#L277?

Reply:
I'm not sure what's the purpose of minusing 1 😅